src/ripple/beast/insight/impl/StatsDCollector.cpp

```diff
@@ -377,9 +375,7 @@ class StatsDCollectorImp
             assert (! s.empty ());
             if (! buffers.empty () && (size + length) > max_packet_size)
             {
-#if BEAST_STATSDCOLLECTOR_TRACING_ENABLED
```

Comment:
Was removing the `BEAST_STATSDCOLLECTOR_TRACING_ENABLED` here intentional?

Reply:
intentional -- my reasoning is that `log()` is already a no-op unless this is defined, so I didn't see the point if the two-level macro check. Feel free to suggest otherwise if you think the check should remain however.